src/octoprint/util/pip.py

```diff
@@ -15,7 +15,9 @@
 import sarge
 
 from octoprint.util import to_unicode
-from octoprint.util.platform import CLOSE_FDS
+
+# from octoprint.util.platform import CLOSE_FDS
+CLOSE_FDS = True
```

Comment:
Could you explain more about that change ?

Reply:
I used the pip file of the octoprint 1.7.3 tag and this also includes this CLOSE_FDS
I need this change here to use the pip file up to the moment when we pull the complete tag. I can revert this change but then it will not be able to run till the merge